packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/registries/add/add.controller.js

```diff
@@ -0,0 +1,40 @@
+export default class AIDashboardAddRegistryCtrl {
+  /* @ngInject */
+  constructor($translate, atInternet, AiDashboardService) {
+    this.$translate = $translate;
+    this.atInternet = atInternet;
+    this.AiDashboardService = AiDashboardService;
+  }
+
+  $onInit() {
+    this.loading = false;
+
+    this.registry = {
+      region: null,
+      username: null,
+      password: null,
+      url: null,
+    };
+  }
+
+  addRegistry() {
+    this.loading = true;
```

Comment:
You should have a reset of the property in a finally block

Reply:
Done :)